src/Moat/Types.hs

```diff
@@ -361,6 +376,11 @@ data Options = Options
     -- }
     -- @
     newtypeTag :: Bool,
+    -- | Newtype declaration syntax to use.
+    --
+    --   This is only meaningful on the Kotlin
+    --   backend.
+    newtypeDeclaration :: NewtypeDeclaration,
```

Comment:
Hmmm!

This comment makes me think that this isn't the right place to put this data.

But I'm also seeing a lot of other fields on the record that share this convention, so I think I will let it slide instead of suggesting a larger refactoring.

Reply:
See #13 for refactoring suggestion